packages/react-components/src/components/MessageThread.tsx

```diff
@@ -847,6 +847,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
       if (!onFetchAttachments || attachment.id in inlineAttachments) {
         return;
       }
+      setInlineAttachments((prev) => ({ ...prev, [attachment.id]: '' }));
```

Comment:
Aha, setting it to empty string is a cool idea, should put a comment here to explain what it is doing

BTW, will empty string be rendered into the img element? Is placeholder still working after you set it to empty string?

Reply:
I believe Josh synced with you offline on this